app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ProjectsController < ApplicationController
+
+  def show
+    @project = Project.find(params[:id])
+    @theme = @project.challenge_theme
+    @count = @project.con_count
+    @exp = @project.average_exp
+  end
+
+  def update
```

Comment:
We wouldn't want to create an update action in our projects controller. Instead you would want to have build a `project_contestants_controller.rb` with a `create` action. Something like this:
```ruby
def create
  @project = Project.find(params[:project_id])
  @contestant = Contestant.find(params[:contestant_id])
  ContestantProject.create!(project: @project, contestant: @contestant)

  redirect_to "/projects/#{@project.id}"
end
```
This is because we aren't updating a `project` or a `contestant` what we are actually doing is creating a new relationship between them. What would happen if you actually had to `update` a project in some way and you have already used your `update` action in here?

So then you would also need to have a route such as:
```ruby
post "/project/:project_id/contestants", to: "project_contestants#create"
```